### DISCUSSION TITLE
Add icons to specific directories

### DISCUSSION THREAD

**Discussion by Khorea1** [gokcehan/lf#1428] [Q&A] [answered]
Hi, I would like to know if it is possible to add icons to specific directories, as shown below

![Captura de tela_2023-09-17_14-59-46](https://github.com/gokcehan/lf/assets/141942237/38469433-ec2a-4af4-9041-c3a922606f89)

**Comment by gokcehan** [comment, marked as answer]
@Khorea1 Yes, this should be possible. See the "colors" and "icons" sections in the documentation.

**Reply by CatsDeservePets** [reply]
@Khorea1 
To add to this reply, it wasn't clear to me until recently that one had to add a `/` at the end of a directory name to make `lf` recognise it correctly inside the `icons` file.
This works: Desktop/ 📁
This doesn't: Desktop 📁
